Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -967,6 +967,16 @@ private OrderResponse HandleUpdateOrderRequest(UpdateOrderRequest request)
 
             ticket.SetOrder(order);
 
+            var hasSufficientBuyingPowerResult = _algorithm.Portfolio.HasSufficientBuyingPowerForOrder([order]);
+
+            if (!hasSufficientBuyingPowerResult.IsSufficient)
+            {
+                var errorMessage = $@"Order Error: id: [{order.Id}], Insufficient buying power to complete order, Reason: {hasSufficientBuyingPowerResult.Reason}.";
```

Comment:
Some order errors only include the order ID, and it is -10. 
It would be helpful to add the Symbol and the Order Time.